src/typecode/data/extensions.yml

```diff
@@ -0,0 +1,2500 @@
+'.1':

```

Comment:
Is there some thirdparty data source you may got this from? We would need to credit them

Reply:
yes, this was taken directly from Github linguist repo.